src/SignalR.Orleans/HostingExtensions.cs

```diff
@@ -14,23 +19,7 @@ public static ISiloBuilder UseSignalR(this ISiloBuilder builder, Action<SignalrO
 
             cfg.ConfigureBuilder?.Invoke(builder, new HostBuilderConfig());
 
-            try
-            {
-                builder.AddMemoryGrainStorage(Constants.PUBSUB_PROVIDER);
-            }
-            catch
-            {
-                /** PubSubStore was already added. Do nothing. **/
-            }
-
-            try
-            {
-                builder.AddMemoryGrainStorage(Constants.STORAGE_PROVIDER);
```

Comment:
should you also remove this? -- i believe if you remove this is a bit "breaking" -- or perhaps  add doc to also register this (as it has custom storage name unlike pubsub)

Reply:
Yeah... I should add that but as it is internal, the user should only tell the name on the options and we register that ourselves...

Reply:
Yes, but either internal we need to register or the consumer needs too if no one does, there won't be any provider registered I guess

Also, just a note - even if its internal the consumer should be able to register it to use a custom grain storage provider

Reply:
I had a second thought and brought back the old code to register the provider.

I need to think better about this change. Perhaps we should change the grain to use the Storage APIs which are based on Facets subsystem in Orleans. 

For now, lets keep as is. Need to research more and will push another PR. Thnanks!